plugin/build.gradle

```diff
@@ -209,16 +209,20 @@ List<String> jacocoExclusions = [
         'org.opensearch.ml.indices.MLInputDatasetHandler',
         'org.opensearch.ml.plugin.*',
         'org.opensearch.ml.task.MLTaskDispatcher',
-        'org.opensearch.ml.task.MLTaskRunner',
-        'org.opensearch.ml.task.MLTrainingTaskRunner',
         'org.opensearch.ml.task.MLPredictTaskRunner',
         'org.opensearch.ml.rest.RestMLTrainingAction',
         'org.opensearch.ml.rest.RestMLPredictionAction',
         'org.opensearch.ml.utils.RestActionUtils',
         'org.opensearch.ml.task.MLTaskCache',
-        'org.opensearch.ml.task.MLTaskManager',
-        'org.opensearch.ml.task.MLTrainAndPredictTaskRunner',
-        'org.opensearch.ml.rest.AbstractMLSearchAction*'
+        'org.opensearch.ml.rest.AbstractMLSearchAction*',
+        'org.opensearch.ml.utils.MLNodeUtils', //0.5
```

Comment:
what is // 0.5 mean? coverage for the class?

Reply:
Yes, we need to add more test to these class